app/assets/javascripts/quotingPanel.js

```diff
@@ -85,13 +114,58 @@ class QuotingPanel {
   addListeners() {
     document.getElementById("SubmitQuote").addEventListener('submit', (event) => { event.preventDefault() });
     document.getElementById("searchAddressButton").addEventListener('click', this.handleSearchAddressClick);
+    document.getElementsByName("serviceExpeditionCost").forEach((element) => { 
+      element.addEventListener('click', (event) => { 
+        const serviceExpeditionCost = Number.parseFloat(event.target.value);
+        const serviceExpeditionDetails = event.target.dataset.label;
+        const serviceExpeditionTime = event.target.id;
```

Comment:
Rather than using the `id` to store the expedition time, I think I would put it in a `data` attribute instead

Reply:
done